### DISCUSSION TITLE
Social Profiles are not shown GoogleSERP Right Hand Side

### DISCUSSION THREAD

**Discussion by venkatboosa** [schemaorg/schemaorg#3118] [General]
Hi,
We have implemented schema.org on our website but we are unable to see updated social profiles on GoogleSERP Right Hand Side.

Kindly suggest us.

Regards,
Venkat

**Comment by WeaverStever** [comment]
Your question is off-topic here, but here is a pointer.

The right hand panel on Google search is called a Knowledge Panel.

If you or you client is a business or localbusiness try https://www.google.com/business/.

For people and artists, Google decides who to include, having a Wikipedia article is generally the pre-qualifier.

Regards,

**Comment by danbri** [comment]
This is not a Google forum. Please ask at Google or on Q/A sites about Google.
